src/content/pages/beginners-day.mdx

```diff
@@ -9,22 +9,20 @@ subtitle:
 
 On **Saturday 19th July**, we'll be hosting a day to help introduce people to programming and its applications, which will feature three tracks running in parallel. The events are designed to welcome newcomers to the Python ecosystem, including a series of talks and panels by junior developers and two workshops designed to introduce complete beginners to web development and data science.
 
-EuroPython Beginners' Day is open to everyone, from high school students to late career changers. Entry is free for EuroPython ticket holders (any ticket type), and €5 otherwise. Tickets for the Beginners' Day will soon be available to be purchased through our tickets page.
-
-## How to sign up for the Beginners' day and choose your track?
-Applications are open until May 11th, and successful applicants will be notified by May 18th. Entry is free for EuroPython ticket holders and €5 otherwise.
-
-The number of places available for each track is as follows:
-* **Beginners' Day Unconference**: 40 places
-* **Django Girls**: 40 places
-* **Humble Data**: 18 places
-
+We are running the following three tracks:
+* **Beginners' Day Unconference (48 places)**: a series of panels and discussions designed to help people new to or early in tech to start and grow their career
+* **Django Girls (40 places)**: a hands-on workshop teaching the basics of web development
+* **Humble Data (18 places)**: a hands-on workshop teaching the basics of data science.
 
+Please read more about each event below.
 
+To participate in the Beginners' Day, **please complete the following steps**:
+* Please sign for your desired track using the form below. Places for each track are limited, and will be assigned on a first-come, first-serve basis.
+* Wait for a confirmation email letting you know which track you have been assigned to.
+* After receiving your confirmation email, please make sure you have a ticket for the event. If you already hold a ticket for EuroPython (any type) you may use this ticket to attend; otherwise, please purchase a ticket for the track of your choice through the ticket store. Tickets are €5 for the Beginners' Day Unconference and Humble Data, and free for Django Girls.
 
 <Button url="https://forms.gle/6mGPPEFMrLsxqHJx5">Apply for Beginners' Day!</Button>
 
-
 ## Beginners' Day Unconference
```

Comment:
At the bottom of this section, would you be able to add a link to the program? The URL should be https://ep2025.europython.eu/beginners-day-unconference once #1267 is merged.